src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue16561.cs

```diff
@@ -23,14 +23,10 @@ public Issue16561(TestDevice device) : base(device)
 
 		[Test]
 		[Category(UITestCategories.Label)]
+		// https://github.com/dotnet/maui/issues/17435
+		[FailsOnMac]
```

Comment:
Can we move that URL into the comment for the attribute? 

```suggestion
		[FailsOnMac("https://github.com/dotnet/maui/issues/17435")]
```